### ISSUE TITLE
Firefox Browser v63+ showing support for audiooutput as part of enumerateDevices() but unavailable as part of the API method call

### ISSUE THREAD

**Issue by avj2352** [mdn/sprints#1691]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [X] Correction or update

# Details

Currently, as per the documentation in Media​Devices​.enumerate​Devices() of MDN, we see that version 63 and onwards - audio-output will be supported by as part Media Devices list provided by the enumerateDevice() method call.

> Please refer to the image: **firefox-browser-compatibility-audiooutput**

When running enumerate-devices() on firefox version 67.0 I am still unable to find audio-output as an option:  

> Please refer to the image: **firefox-enumerateDevices.png**

# Conclusion

Request to either update the documentation site with the revised version in which this feature will be supported OR fix the audio-output device as part of the enumerateDevices() method implementation.

![firefox-browser-compatibility-audiooutput](https://user-images.githubusercontent.com/6997529/59039612-c6988d80-8892-11e9-90f9-1834854bc189.PNG)

![firefox-enumerateDevices](https://user-images.githubusercontent.com/6997529/59039613-c7c9ba80-8892-11e9-8495-2f062bcd40fe.PNG)

**Comment by avj2352** [comment]
Currently, the latest version of the Chrome browser is able to support **audiooutput** device as part of **enumerateDevices()** method. Please find the attached screenshot


Chrome version 74.0.x
![chrome-enumerate-devices](https://user-images.githubusercontent.com/6997529/59040379-2fccd080-8894-11e9-8aac-86338db899f6.PNG)

**Comment by avj2352** [comment]
For comparison, I am attaching Javascript codesnippet code. (Run both on chrome and firefox). The screenshot were taken against the below code snippet

```js
navigator.mediaDevices.enumerateDevices()
    .then(function(devices) {
        const subTitleDom = document.querySelector('#subtitle');
        subTitleDom.innerHTML = 'List of Media Elements';
        // Iterating through the list
        devices.forEach(function(device) {
            let elem = document.createElement("li");
            let [kind, type, direction] = device.kind.match(/(\w+)(input|output)/i);
            elem.innerHTML = "<strong>" + device.label + "</strong> (" + direction + ")" + " --- " + type + "/" + device.deviceId;
            // Check if audio - populate input and output devices
            if (type === "audio") {
                audioList.appendChild(elem);
            } else if (type === "video") {
                audioList.appendChild(elem);
            }
        });
    });
```

**Comment by SphinxKnight** [comment]
@avj2352 This has been fixed by https://github.com/mdn/browser-compat-data/pull/4788 / https://github.com/mdn/browser-compat-data/commit/91b3f361d51b203d9c1b1ad7e8bf3f83ade0041d where the data has been updated. It now shows that a preference (`media.setsinkid.enabled`) must be set to `true` to enumerate outputs.

The compatibility table of the MDN page is up to date as far as I can tell on my side.

I've tested the code sample of the page and the preference actually is necessary (Fx 70)
